lang/values.md

```diff
@@ -271,10 +271,18 @@ Note that none of the definitions in this section are needed to define the seman
 They only serve as internal consistency requirements of the semantics.
 It would be a specification bug if the representation relations defined above violated these properties.
 
+```rust
+trait DefinedRelation {
+  /// returns whether `self` is less or as defined as `other`
+  fn le_defined(&self, other: &Self) -> bool;
```

Comment:
We've mostly been using 4-space indent I think, let's stick to that.

Reply:
whoops!